mars/contrib/dask/scheduler.py

```diff
@@ -30,22 +30,26 @@ def mars_scheduler(dsk: dict, keys: List[List[str]]):
     ----------
     dsk: Dict
         Dask graph, represented as a task DAG dictionary.
-    keys: List[List[str]]
-        2d-list of Dask graph keys whose values we wish to compute and return.
+    keys: Union[List[List[str]], List[str]]
+        1d or 2d list of Dask graph keys whose values we wish to compute and return.
 
     Returns
     -------
     Object
-        Computed values corresponding to the provided keys.
+        Computed values corresponding to the provided keys with same dimension.
     """
-    res = reduce(mars_dask_get(dsk, keys)).execute().fetch()
-    if not isinstance(res, List):
-        return [[res]]
-    else:
-        return res
 
+    if isinstance(keys, List) and not isinstance(keys[0], List):  # 1d keys
+        return map(lambda x: x.execute().fetch(), mars_dask_get(dsk, keys))
```

Comment:
Looks like it will trigger many Mars execution, `mars.execute([t1, t2, t3...]` could batch run multiple tileables. Is it possible to submit multiple tileables at one time?

Reply:
Sure, `mars.execute` way is more efficient.